docs/source/using/envs.rst

```diff
@@ -216,3 +216,38 @@ Verify that the new environment was installed correctly:
    conda list
 
 Next, we'll take a look at managing Python.
```

Comment:
Is this line still correct? I don't understand what it meant at the end of the file.